File: src/SOP8/Base/Account/Base.Account.IBLL/Request/RequestNewUser.cs
namespace Base.Account.IBLL.Request
{
    public class RequestNewUser
    {
        private int? m_regularMemberNo = null;
        private string m_strUserID = "";
        private string m_strNickName = "";
        private int m_nGrade = -1;
        private int? m_siteNo = null;
        private string m_strMemo = null;

        public int? RegularMemberNo
        {
            get { return m_regularMemberNo; }
            set { m_regularMemberNo = value; }
        }

        public string UserID
        {
            get { return m_strUserID; }
            set { m_strUserID = value; }
        }

        public string NickName
        {
            get { return m_strNickName; }
            set { m_strNickName = value; }
        }

        public int Grade
        {
            get { return m_nGrade; }
            set { m_nGrade = value; }
        }

        public int? SiteNo
        {
            get { return m_siteNo; }
            set { m_siteNo = value; }
        }

        public string Memo
        {
            get { return m_strMemo; }
            set { m_strMemo = value; }
        }
    }
}

File: src/SOP8/광양/EgovLogin/base-account-ibll/src/main/java/egovframework/base/account/ibll/request/RequestNewUser.java
package egovframework.base.account.ibll.request;

public class RequestNewUser {

    private Integer regularMemberNo = null;
    private String userID = "";
    private String nickName = "";
    private int grade = -1;
    private Integer siteNo = null;
    private String memo = null;

    public Integer getRegularMemberNo() {
        return regularMemberNo;
    }

    public void setRegularMemberNo(Integer regularMemberNo) {
        this.regularMemberNo = regularMemberNo;
    }

    public String getUserID() {
        return userID;
    }

    public void setUserID(String userID) {
        this.userID = userID;
    }

    public String getNickName() {
        return nickName;
    }

    public void setNickName(String nickName) {
        this.nickName = nickName;
    }

    public int getGrade() {
        return grade;
    }

    public void setGrade(int grade) {
        this.grade = grade;
    }

    public Integer getSiteNo() {
        return siteNo;
    }

    public void setSiteNo(Integer siteNo) {
        this.siteNo = siteNo;
    }

    public String getMemo() {
        return memo;
    }

    public void setMemo(String memo) {
        this.memo = memo;
    }
}